### PR TITLE
app wide font styling

### LINKED ISSUE #147: App wide font styling
**Is your feature request related to a problem? Please describe.**
This feature will style all of the fonts, their weight and spacing.

### PR DESCRIPTION
**Pull request by gonzalespaulb**
## Changes
_List Changes Introduced by this PR_
1. Font Color
2. Font Weight
3. Font Spacing

## Purpose
More pizzazz

Closes #147

### CODE REVIEW

**Review by krferrell — APPROVED** [review]
Great work updating the styles! Looking so good now.

File: pokeapp/src/index.css [hunk 1]
```diff
@@ -10,7 +10,7 @@ body {
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
-}
+} */
```
Comment (Lap343) [inline pokeapp/src/index.css:13]: is this still needed commented out?

File: pokeapp/src/components/SidebarNav.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ export const SidebarNav = ({ setIsBadgeSideBar, setBadgeBtnActive1, setBadgeBtnA
           setIsBadgeSideBar(true);
         }}
       >
-        Badges
+        {/* {badgeBtnActive1 ? "sidebar-nav-font-1" : "sidebar-nav-font-2"} */}
```
Comment (Lap343) [inline pokeapp/src/components/SidebarNav.js:15]: Do you need this comment?

**Review by Lap343 — APPROVED** [review]
Code looks good

**Pull request merged**